src/settings/index.ts

```diff
@@ -96,6 +96,11 @@ export const IFRAME_TEMPLATES = [
   VenueTemplate.viewingwindow,
 ];
 
+export const EMBEDDABLE_CONTENT_TEMPLATES = [
+  ...IFRAME_TEMPLATES,
+  ...ZOOM_URL_TEMPLATES,
+];
+
```

Comment:
Maybe this `Object.freeze(EMBEDDABLE_CONTENT_TEMPLATES );` is needed for "better safe than sorry", but also one more for each of those `_TEMPLATES` spread inside it.